contract/src/main/java/com/ritense/valtimo/contract/form/FormFieldDataResolver.java

```diff
@@ -30,6 +32,8 @@ default boolean supports(String externalFormFieldType) {
         return supports(ExternalFormFieldType.fromKey(externalFormFieldType));
     }
 
+    @Deprecated(forRemoval = true, since = "9.21")
     Map<String, Object> get(String documentDefinitionName, UUID documentId, String... varNames);
 
+    Map<String, Object> get(String documentDefinitionName, UUID documentId, JsonNode formDefinition, String... varNames);
```

Comment:
If you give this method a default implementation (e.g. just call the old get) it won't be a breaking change.